.travis.yml

```diff
@@ -56,7 +56,6 @@ notifications:
           - http://julia.mit.edu:8000/travis-hook
 before_install:
     - make check-whitespace
-    - JULIA_SYSIMG_BUILD_FLAGS="--output-ji ../usr/lib/julia/sys.ji"
```

Comment:
Right, when the `--precompiled=no` configuration changed forms to being bundled inside sys.dll in a default build, the remaining purpose of this test was then only about the no-linker build_sysimg use case.